tutorials/reprojection.ipynb

```diff
@@ -48,9 +61,9 @@
    "id": "199e7425",
    "metadata": {},
    "source": [
-    "## Local Dask Cluster\n",
+    "## Preparation: Create a local Dask cluster\n",
```

Comment:
It might be good to briefly explain why we are using a local Dask cluster as a basis for our examples. I would also be able to do reprojections without Dask, right?